crt_portal/cts_forms/migrations/0085_remove-assignee_20200716.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.13 on 2020-07-16 15:26
+
+from django.db import migrations
+from datetime import datetime
+import pytz
+from actstream import action
+from django.contrib.auth import get_user_model
+from actstream import registry
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0084_form_template_standardization'),
+    ]
+
+    def remove_assignee(apps, schema_editor):
+        User = get_user_model()
+        userlist = User.objects.all().order_by('id')
+        # // Assuming that first user is the root user or superuser. If there are no users, skip assignee removal
+        if len(userlist) > 0:
+            superuser = userlist[0]
+            # get all reports
+            reports = apps.get_model('cts_forms', 'Report')
+            # Register Report to be streamed
+            registry.register(reports)
+            timezone = pytz.timezone("UTC")
+            july1 = timezone.localize(datetime(2020, 7, 1))
+            for report in reports.objects.all():
+                # remove assignee for report closed date is before june 30 2020,
+                if report.closed_date is not None:
+                    print(report.closed_date)
+                    if report.closed_date < july1:
+                        report.assigned_to = None
+                        report.save()
+                        # Add the closed activity to activity stream
+                        action.send(superuser, verb='Assignee Removed: ',
+                                    description='Removed assignee for closed record before July 1, 2020',
+                                    target=report)
+    operations = [
+        migrations.RunPython(remove_assignee),
```

Comment:
Do we want this migration to be reversed? Even if not, I'd still suggest throwing in a dummy reverse migration so we can roll back when needed (easier for local dev imo).

Reply:
@cantsin, can we use reverting migration by running previous migrations?. Read that this is one option, don't know if you have any good/bad experience?